hapi-fees/src/test/java/com/hedera/services/usage/token/TokenGetNftInfoUsageTest.java

```diff
@@ -34,7 +34,7 @@
 import static com.hederahashgraph.fee.FeeBuilder.LONG_SIZE;
 import static org.junit.Assert.assertEquals;
 
-public class TokenGetNftInfoUsageTest {
+class TokenGetNftInfoUsageTest {
 	String memo = "Hope";
 	NftID id = IdUtils.asNftID("0.0.75231", 1);
```

Comment:
Was fixed with more changes in `master`
https://github.com/hashgraph/hedera-services/pull/1762/commits/8b1a68faf5050f6d71345b04c5a4f52aa47331b0